Linux Kernel 5.0 support possible?

/etc/kernel/postinst.d/dkms:
ERROR (dkms apport): binary package for rtl88x2bu: 5.3.1 not found
Error! Bad return status for module build on kernel: 5.0.0-050000-generic (x86_64)
Consult /var/lib/dkms/rtl88x2bu/5.3.1/build/make.log for more information.


DKMS make.log for rtl88x2bu-5.3.1 for kernel 5.0.0-050000-generic (x86_64)
Sa 9. Mär 10:59:45 CET 2019
make ARCH=x86_64 CROSS_COMPILE= -C /lib/modules/5.0.0-050000-generic/build M=/var/lib/dkms/rtl88x2bu/5.3.1/build  modules
make[1]: Verzeichnis „/usr/src/linux-headers-5.0.0-050000-generic“ wird betreten
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_cmd.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_security.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_debug.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_io.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ioctl_query.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ioctl_set.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ieee80211.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_mlme.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_mlme_ext.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_mi.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_wlan_util.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_vht.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_pwrctrl.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_recv.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_sta_mgt.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ap.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/mesh/rtw_mesh.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/mesh/rtw_mesh_pathtbl.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/mesh/rtw_mesh_hwmp.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_xmit.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_p2p.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rson.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_tdls.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_br_ext.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_iol.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_sreset.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_btcoex_wifionly.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_btcoex.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_beamforming.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_odm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rm_fsm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/efuse/rtw_efuse.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/osdep_service.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/os_intfs.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/usb_intf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/usb_ops_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/ioctl_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/xmit_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/mlme_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/recv_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/ioctl_cfg80211.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_cfgvendor.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/wifi_regd.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_android.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_proc.o
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/rtw_android.c: In function ‘rtw_android_priv_cmd’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/rtw_android.c:660:62: error: macro "access_ok" passed 3 arguments, but takes just 2
  if (!access_ok(VERIFY_READ, priv_cmd.buf, priv_cmd.total_len)) {
                                                              ^
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/rtw_android.c:660:7: error: ‘access_ok’ undeclared (first use in this function)
  if (!access_ok(VERIFY_READ, priv_cmd.buf, priv_cmd.total_len)) {
       ^~~~~~~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/rtw_android.c:660:7: note: each undeclared identifier is reported only once for each function it appears in
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_rhashtable.o
scripts/Makefile.build:276: recipe for target '/var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_android.o' failed
make[2]: *** [/var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_android.o] Error 1
make[2]: *** Auf noch nicht beendete Prozesse wird gewartet …
Makefile:1553: recipe for target '_module_/var/lib/dkms/rtl88x2bu/5.3.1/build' failed
make[1]: *** [_module_/var/lib/dkms/rtl88x2bu/5.3.1/build] Error 2
make[1]: Verzeichnis „/usr/src/linux-headers-5.0.0-050000-generic“ wird verlassen
Makefile:1995: recipe for target 'modules' failed
make: *** [modules] Error 2

**-->> Confirmed build, testing since 30 minutes...
-> confirmed working!**

_Thanks cilynx for the fast fix._



Here's the make log. Found error warning about ISO C90 forbids variable lenght array 'Input'. Hopefully nothing bad ?

<pre><span style="background-color:#FFFFFF"><font color="#3F3F3F">  GNU nano 2.9.3    /var/lib/dkms/rtl88x2bu/5.3.1/5.0.0-050000-generic/x86_64/log/make.log                </font></span>

DKMS make.log for rtl88x2bu-5.3.1 for kernel 5.0.0-050000-generic (x86_64)
Mi 13. Mär 12:36:54 CET 2019
make ARCH=x86_64 CROSS_COMPILE= -C /lib/modules/5.0.0-050000-generic/build M=/var/lib/dkms/rtl88x2bu/5.3.1/build  modules
make[1]: Verzeichnis „/usr/src/linux-headers-5.0.0-050000-generic“ wird betreten
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_cmd.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_security.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_debug.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_io.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ioctl_query.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ioctl_set.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ieee80211.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_mlme.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_mlme_ext.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_mi.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_wlan_util.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_vht.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_pwrctrl.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_recv.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_sta_mgt.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_ap.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/mesh/rtw_mesh.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/mesh/rtw_mesh_pathtbl.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/mesh/rtw_mesh_hwmp.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_xmit.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_p2p.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rson.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_tdls.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_br_ext.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_iol.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_sreset.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_btcoex_wifionly.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_btcoex.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_beamforming.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_odm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_rm_fsm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/efuse/rtw_efuse.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/osdep_service.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/os_intfs.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/usb_intf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/usb_ops_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/ioctl_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/xmit_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/mlme_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/recv_linux.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/ioctl_cfg80211.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_cfgvendor.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/wifi_regd.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_android.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_proc.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/rtw_rhashtable.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/os_dep/linux/ioctl_mp.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_intf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_com.o
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_write_reg’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:46:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length + 1];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_read_reg’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:132:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length + 1];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_write_rf’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:266:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_read_rf’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:308:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_rate’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:417:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length + 1];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_channel’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:467:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length + 1];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_ch_offset’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:496:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length + 1];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_bandwidth’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:524:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_txpower_index’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:556:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length + 1];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_txpower’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:585:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_ant_tx’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:623:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length + 1];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_ant_rx’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:671:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length + 1];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_disable_bt_coexist’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:851:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8 input[wrqu->data.length + 1];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_arx’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:889:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_pwrtrk’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1068:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_psd’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1106:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length + 1];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_dump’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1221:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_phypara’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1247:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_SetRFPath’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1273:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_switch_rf_path’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1322:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->length];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_QueryDrv’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1371:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  char input[wrqu->data.length];
  ^~~~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_PwrCtlDM’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:1400:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->length];
  ^~
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c: In function ‘rtw_mp_hwtx’:
/usr/src/rtl88x2bu-5.3.1/os_dep/linux/ioctl_mp.c:2050:2: warning: ISO C90 forbids variable length array ‘input’ [-Wvla]
  u8  input[wrqu->data.length];
  ^~
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_com_phycfg.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_phy.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_dm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_dm_acs.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_btcoex_wifionly.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_btcoex.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_mp.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_mcc.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_hci/hal_usb.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/led/hal_led.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/led/hal_usb_led.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/hal_halmac.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/rtl8822b_halinit.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/rtl8822b_mac.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/rtl8822b_phy.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/rtl8822b_cmd.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/rtl8822b_ops.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/hal8822b_fw.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/usb/rtl8822bu_halinit.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/usb/rtl8822bu_halmac.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/usb/rtl8822bu_io.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/usb/rtl8822bu_xmit.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/usb/rtl8822bu_recv.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/usb/rtl8822bu_led.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/rtl8822b/usb/rtl8822bu_ops.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/efuse/rtl8822b/HalEfuseMask8822B_USB.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_api.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_bb_rf_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_cfg_wmac_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_common_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_efuse_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_flash_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_fw_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_gpio_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_init_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_mimo_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_pcie_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_sdio_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_usb_88xx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_cfg_wmac_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_common_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_gpio_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_init_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_pcie_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_phy_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_pwr_seq_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_sdio_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/halmac/halmac_88xx/halmac_8822b/halmac_usb_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_debug.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_antdiv.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_soml.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_smt_ant.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_antdect.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_interface.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_phystatus.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_hwconfig.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_dig.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_pathdiv.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_rainfo.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_dynamictxpower.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_adaptivity.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_cfotracking.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_noisemonitor.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_beamforming.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_dfs.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/txbf/halcomtxbf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/txbf/haltxbfinterface.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/txbf/phydm_hal_txbf_api.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_adc_sampling.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_ccx.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_psd.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_primary_cca.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_cck_pd.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_rssi_monitor.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_auto_dbg.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_math_lib.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_api.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/phydm_pow_train.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/halrf/halrf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/halrf/halphyrf_ce.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/halrf/halrf_powertracking_ce.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/halrf/halrf_powertracking.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/halrf/halrf_kfree.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/rtl8822b/halhwimg8822b_bb.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/rtl8822b/halhwimg8822b_mac.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/rtl8822b/halhwimg8822b_rf.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/halrf/rtl8822b/halrf_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/rtl8822b/phydm_hal_api8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/halrf/rtl8822b/halrf_iqk_8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/rtl8822b/phydm_regconfig8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/rtl8822b/phydm_rtl8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/phydm/txbf/haltxbf8822b.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/btc/halbtc8822bwifionly.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/btc/halbtc8822b1ant.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/hal/btc/halbtc8822b2ant.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/platform/platform_ops.o
  CC [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/core/rtw_mp.o
  LD [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/88x2bu.o
  Building modules, stage 2.
  MODPOST 1 modules
  CC      /var/lib/dkms/rtl88x2bu/5.3.1/build/88x2bu.mod.o
  LD [M]  /var/lib/dkms/rtl88x2bu/5.3.1/build/88x2bu.ko
make[1]: Verzeichnis „/usr/src/linux-headers-5.0.0-050000-generic“ wird verlassen
<span style="background-color:#FFFFFF"><font color="#3F3F3F"></font></span>

--- a/os_dep/linux/rtw_android.c
+++ b/os_dep/linux/rtw_android.c
@@ -657,7 +657,11 @@ int rtw_android_priv_cmd(struct net_device *net, struct ifreq *ifr, int cmd)
 		goto exit;
 	}
 
+	#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 0, 0))
+	if (!access_ok(priv_cmd.buf, priv_cmd.total_len)) {
+	#else
 	if (!access_ok(VERIFY_READ, priv_cmd.buf, priv_cmd.total_len)) {
+	#endif
 		RTW_INFO("%s: failed to access memory\n", __FUNCTION__);
 		ret = -EFAULT;
 		goto exit;
@@ -926,7 +930,7 @@ int rtw_android_priv_cmd(struct net_device *net, struct ifreq *ifr, int cmd)
 		bytes_written = rtw_android_set_aek(net, command, priv_cmd.total_len);
 		break;
 #endif
-	
+
 	case ANDROID_WIFI_CMD_DRIVERVERSION: {
 		bytes_written = strlen(DRIVERVERSION);
 		snprintf(command, bytes_written + 1, DRIVERVERSION);

